### ISSUE TITLE
[BUG] Unable to set avatar of another profile even with the necessary permission

### ISSUE THREAD

**Issue by davidshugert** [RocketChat/Rocket.Chat#11334]
### Description:
I'm running a small script that updates all RC users avatars from a database. When I fetch the endpoint I get a success message, but the avatar doesn't get updated.

It looks like the server doesn't enter [here](https://github.com/RocketChat/Rocket.Chat/blob/2e38466800edf158d0117bf9cbdc8d1c9568ebf9/packages/rocketchat-lib/server/functions/setUserAvatar.js#L32) or if it does it is not accepting raw Buffers. I have tried changing the avatar with a hosted image (body containing userId and avatarUrl) as in [this example ](https://rocket.chat/docs/developer-guides/rest-api/users/setavatar/#example-call-1)and it does work. The problem is with an image that has been uploaded. I followed the example in the official [RocketChat documentation](https://rocket.chat/docs/developer-guides/rest-api/users/setavatar/#example-call). Nevertheless, looking at the testing folders I found that this option is not tested. The only function tested in Set Avatar is with a [photo from a URL](https://github.com/RocketChat/Rocket.Chat/blob/2e38466800edf158d0117bf9cbdc8d1c9568ebf9/tests/end-to-end/api/01-users.js#L106).

#### Code Used:
```javascript
  const form = new FormData()
  examplePhoto = fs.readFileSync('./example.jpg') //Local image
  form.append('userId', userId) // ID of the user
  form.append('image', examplePhoto) //JPEG file in buffer

  let responseStream = await fetch(`${rocketChatUrl}/api/v1/users.setAvatar`, {
    method: 'POST',
    headers,
    body: form
  })
console.log(responseStream.json()) //Output: {Success: true}
```

### Expected behavior:
When the 'user.setAvatar' endpoint is reached, and a local file is uploaded, the server must send a 'success: true' and the user's avatar will be updated.

### Actual behavior:
When the user.setAvatar endpoint is reached as a POST method with all the parameters required, the server responds with an success 'true' response but the uploaded image file is not actually changed on the user it is being applied to.

### Server Setup Information:

- Version of Rocket.Chat Server: 2.10.2
- Official Rocket Chat image
- Operating System: Ubuntu 17.10
- NodeJS Version: 9.11

**Comment by MarcosSpessatto** [comment]
Hey @davidshugert, I ran your code snippet and it worked perfectly. I just have a question, regarding the variable `userId` of your code, are you trying the avatar of your profile or another profile?

**Comment by davidshugert** [comment]
Hello @MarcosSpessatto , I'm trying to change the Avatar on another profile but I am  authenticated as admin.

**Comment by MarcosSpessatto** [comment]
Okay, so we have a bug here, I'll fix it, thanks for letting us know

**Comment by 0xbaadf00d** [comment]
Any estimated time when this issue will be fixed ? (running on version 0.70.3)
